### PR TITLE
LUN-147: Use Theming for Fonts

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/constants/theme/fonts.ts [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
-  contentFontRegular: 'SourceSansPro-Regular'
+  contentFontRegular: 'SourceSansPro-Regular',
+  defaultFontWeight: 600,
+  lightFontWeight: 400,
+  capsLetterSpacing: '0.4px',
+  listTitleLetterSpacing: '0.11px',
+  smallFontSize: `${wp('3%')}px`,
+  defaultFontSize: `${wp('4%')}px`,
+  largeFontSize: `${wp('4.5%')}px`,
+  headingFontSize: `${wp('5%')}px`
```
Comment (Taggotty) [inline src/constants/theme/fonts.ts:13]: couldn't really come up with  a better naming here. I am open for any ideas

Comment (SteffiStuffel) [reply src/constants/theme/fonts.ts:13]: i think heading is fine, but if you are unhappy anything else i have in mind is extraLarge

File: src/routes/ProfessionSubcategoryScreen.tsx [hunk 2]
```diff
@@ -67,6 +67,12 @@ const ProfessionSubcategoryScreen = ({ route, navigation }: ProfessionSubcategor
   const [selectedId, setSelectedId] = useState<number | null>(null)
   const { data: disciplines, error, loading } = useLoadDisciplines(module)
 
+  useFocusEffect(
```
Comment (SteffiStuffel) [inline src/routes/ProfessionSubcategoryScreen.tsx:70]: Why is this needed?
